src/DefaultBuilder/src/WebHost.cs

```diff
@@ -262,7 +260,10 @@ internal static void ConfigureWebDefaultsCore(IWebHostBuilder builder)
             services.AddTransient<IStartupFilter, ForwardedHeadersStartupFilter>();
             services.AddTransient<IConfigureOptions<ForwardedHeadersOptions>, ForwardedHeadersOptionsSetup>();
 
-            services.AddRoutingCore();
+            if (configureRouting == default)
```

Comment:
== default?

Reply:
Matter of preference as I don't think it makes a difference in this case. If `default` and generics had existed in C# from the very beginning, I wonder which would have been the preferred default for checking whether a value has been populated.

Reply:
We should be using is null here.